src/gsi/SolidPropertiesSteadyState.cpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * @file SolidPropertiesSteadyState.cpp
+ *
+ * @brief SolidProperties class when solid properties are required
+ *        for steady state pyrolysis and solid conduction.
+ */
+
+/*
+ * Copyright 2018 von Karman Institute for Fluid Dynamics (VKI)
+ *
+ * This file is part of MUlticomponent Thermodynamic And Transport
+ * properties for IONized gases in C++ (Mutation++) software package.
+ *
+ * Mutation++ is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Mutation++ is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with Mutation++.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+#include "AutoRegistration.h"
+#include "Utilities.h"
+
+#include "SolidProperties.h"
+
+using namespace Mutation::Utilities::Config;
+
+namespace Mutation {
+    namespace GasSurfaceInteraction {
+
+class SolidPropertiesSteadyState : public SolidProperties
+{
+public:
+    /**
+     * Default Constructor
+     */
+    SolidPropertiesSteadyState(ARGS args)
+        : SolidProperties(args),
+          m_phi(1.),
+          m_h_v(0.)
+    {
+            if (args.s_node_solid_props.tag() == "solid_properties") {
```

Comment:
please fix indentation

Reply:
I think I fixed it, can you recheck please